k8s/kong_monitoring.tf

```diff
@@ -1,13 +1,77 @@
-resource "kubernetes_manifest" "kong_monitoring_plugin" {
```

Comment:
@witnsby hey, Alex, what do you think do we really need this file, or can we place it just in kong setup files? 
For me, this seems related to kong.

Reply:
it up to you

Reply:
@dumikau please, as author decide :D